lib/sycamore/sycamore/connectors/file/file_writer_ray.py

```diff
@@ -39,6 +39,8 @@ def on_write_start(self) -> None:
         is_s3_uri = parsed_uri.scheme == "s3"
 
         if not is_s3_uri and self._filesystem.get_file_info(self._root).type is FileType.NotFound:
+            if self._root == "":
+                self._root = "./"
```

Comment:
Can we put this logic in the __init__ method? I'd just feel more comfortable keeping state as immutable as possible.

Reply:
works for me!